go.mod

```diff
@@ -1,23 +1,18 @@
 module github.com/tozny/utils-go
 
 require (
+	cloud.google.com/go v0.37.4 // indirect
 	github.com/aws/aws-sdk-go v1.19.8
-	github.com/fortytw2/leaktest v1.3.0 // indirect
 	github.com/go-pg/pg v8.0.3+incompatible
 	github.com/google/go-cmp v0.3.0 // indirect
 	github.com/google/uuid v1.1.0
 	github.com/jinzhu/inflection v0.0.0-20180308033659-04140366298a // indirect
 	github.com/kr/pretty v0.1.0 // indirect
 	github.com/mailru/easyjson v0.0.0-20190403194419-1ea4449da983 // indirect
 	github.com/olivere/elastic v6.2.17+incompatible
-	github.com/onsi/ginkgo v1.8.0 // indirect
-	github.com/onsi/gomega v1.5.0 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
 	github.com/robinjoseph08/go-pg-migrations v0.1.2
-	github.com/stretchr/testify v1.3.0 // indirect
-	github.com/tozny/e3db-clients-go v0.0.18
+	github.com/tozny/e3db-clients-go v0.0.19-0.20190507195351-473922bdcefc
```

Comment:
when updating both this and e3db-clients-go, the most painless way is to tag your feature branch with a release tag (v.0.0.20).